frontend/packages/api_client/src/runtime.ts

```diff
@@ -144,7 +144,7 @@ export class Configuration {
     constructor(private configuration: ConfigurationParameters = {}) {}
 
     get basePath(): string {
-        return this.configuration.basePath || BASE_PATH;
+        return this.configuration.basePath != null ? this.configuration.basePath : BASE_PATH;
```

Comment:
Was this auto-generated? If so we probably have a version mismatch between my version and yours.

Reply:
@giovannicimolin, it was auto-generated and I verified that I have the latest version of all the packages installed.

Reply:
@giovannicimolin, in `frontend/packages/api_client/.openapi-generator/VERSION`, the version in my PR is newer than the version that is in the `new-console` branch. This might be due to that. If using a newer version is a problem, we should pin the docker image to a known good version and update it when required.

Reply:
No problem, I will update my local image.
As long as it doesn't break the deployments I'm fine with it :smile: